### PR TITLE
[API compatibility]update doc of LayerNorm

### PR DESCRIPTION
**Pull request by LynnLi777**
修改 paddle.nn.functional.LayerNorm 的文档：
1. 增加与torch兼容的接口的说明

关联PR：https://github.com/PaddlePaddle/Paddle/pull/76304

### CODE REVIEW

File: docs/api/paddle/nn/LayerNorm_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/nn/LayerNorm_cn.rst:1]: 既然有关键字参数，是不是得在函数声明里 ``.. py:class:: paddle.nn.LayerNorm(normalized_shape, epsilon=1e-05, weight_attr=None, bias_attr=None, name=None)`` 加个 \*

Comment (LynnLi777) [reply docs/api/paddle/nn/LayerNorm_cn.rst:1]: 已修改

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
